<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BillPayment1</class>
 <widget class="QMainWindow" name="BillPayment1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AJ Bank | Bill Payment</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/background/BillPayment.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="MobileBillB">
    <property name="geometry">
     <rect>
      <x>452</x>
      <y>325</y>
      <width>560</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/MobileBill.png);
background-position:center;
border-radius:100%;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="UtilityBillB">
    <property name="geometry">
     <rect>
      <x>452</x>
      <y>537</y>
      <width>560</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/UtilityBill.png);
background-position:center;
border-radius:100%;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>93</y>
      <width>150</width>
      <height>581</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Back.png);
border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="LogoutButton">
    <property name="geometry">
     <rect>
      <x>949</x>
      <y>384</y>
      <width>421</width>
      <height>384</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Logout.png);
border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SuggestButton">
    <property name="geometry">
     <rect>
      <x>949</x>
      <y>0</y>
      <width>421</width>
      <height>384</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Suggest.png);
border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="AccountNo">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>60</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Blue.png);
font: 25 11pt &quot;Montserrat Thin&quot;;
color: rgb(6, 53, 122);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
